index.html

```diff
@@ -20,19 +23,23 @@
       <script src="https://oss.maxcdn.com/libs/respond.js/1.4.2/respond.min.js"></script>
     <![endif]-->
   </head>
-  <body ng-app="GiphyApp">
-    <div class="container">
+  <body ng-app="GiphyApp" infinite-scroll="Search2()" infinite-scroll-distance="3">
+    <div class="container" ng-controller="GifSearch">
       <h1 class="text-center">Welcome to Giphy Search</h1>
 
-      <form class="form-inline">
+      <form class="form-inline" ng-submit="search(box)">
         <div class="form-group">
           <label for="search">Search</label>
-          <input type="text" placeholder="Search for GIFs" id="search" class="form-control">
+          <input type="text" placeholder="Search for GIFs" id="search" name="box" ng-model="box" class="form-control">
         </div>
-        <button type="submit" class="btn btn-primary">Search</button>
+        <button type="submit" class="btn btn-success">Search</button>
       </form>
 
-      <div id="results"></div>
+      <div id="results">
+        <div class="well" ng-repeat="gif in gifs track by $index">
+          <img src={{gif.images.original.url}} />
+        </div>
+      </div>
     </div>
```

Comment:
Div element to host the infinite scrolling:

    <div class="row" id="results" infinite-scroll="myPagingFunction()" infinite-scroll-distance="3">
        <div class="well col-md-6" ng-repeat="giphy in giphys track by giphy.id">
            <img ng-src="{{giphy.images.fixed_height.url}}" alt="Description" />
            <a ng-href="{{giphy.images.fixed_height.url}}">link</a>
        </div>
    </div>